lib/data/models/token_contract_asset.dart

```diff
@@ -14,11 +14,22 @@ class TokenContractAsset with _$TokenContractAsset {
     required int decimals,
     required Address address,
     required NetworkType networkType,
-    required int version,
+    required TokenWalletVersion version,
     int? chainId,
     String? logoURI,
   }) = _TokenContractAsset;
 
   factory TokenContractAsset.fromJson(Map<String, dynamic> json) =>
       _$TokenContractAssetFromJson(json);
 }
+
+extension IntTokenWalletVersionExtension on int {
```

Comment:
As i can remember, we tried to get rid of extensions on general purpose types.

Reply:
It's but we can't combine 2 different types of parsing json for this enum.
This sucks ext is used in old-db migration + `AssetsService` because backend sends us only this digits.
But we want to store this field as a normal json string with its converter.